backend/src/main/java/com/woowacourse/ternoko/core/application/InterviewService.java

```diff
@@ -119,11 +121,18 @@ private Coach getCoachById(final Long interviewRequest) {
     }
 
     @Transactional(readOnly = true)
-    public InterviewResponse findInterviewResponseById(final Long interviewId) {
+    public InterviewResponse findInterviewResponseById(final Long memberId, final Long interviewId) {
         final Interview interview = getInterviewById(interviewId);
+        validateOwnMember(memberId, interview);
         return InterviewResponse.from(interview);
     }
 
+    private void validateOwnMember(final Long memberId, final Interview interview) {
+        if (!interview.getCoach().isSameId(memberId) && !interview.getCrew().isSameId(memberId)) {
+            throw new InvalidInterviewMemberException(INVALID_INTERVIEW_BY_MEMBER);
+        }
```

Comment:
r: 이런식으로 도메인로직으로 옮겨보면 좋을 것 같아요

```suggestion
        if (!interview.containsMember(memberId)) {
            throw new InvalidInterviewMemberException(INVALID_INTERVIEW_BY_MEMBER);
        }
```

Reply:
말슴하신대로 Interview 내부로직으로 동작하게 수정하였습니다.